core/src/test/java/kafka/test/junit/ClusterTestExtensions.java

```diff
@@ -175,7 +175,7 @@ private void processClusterTest(ExtensionContext context, ClusterTest annot, Clu
             case DEFAULT:
                 autoStart = defaults.autoStart();
                 break;
-            default:
+            default :
```

Comment:
This contradicts
```
 <setting id="org.eclipse.jdt.core.formatter.insert_space_before_colon_in_case" value="do not insert"/>
```
in `eclipse-formatter.xml`.

Reply:
Yes, I found that the eclipse formatter handles the default case independely. Additional configuration added to fix this. :+1: